skills/_artifacts/skill_spec.md

```
@@ -0,0 +1,127 @@
+| input-handling | core | input-handling | setState-on-change + setUrl-on-blur/debounce | 3 |
+| nextjs-ssr | framework | nextjs-integration | searchParams forwarding, hydration, Proxy/layout | 5 |
+| react-router-remix-setup | framework | router-integration | rr6/rr7/remix imports and NavigateOptions | 2 |
+| form-library-integration | composition | form-composition | useUrlState + react-hook-form pattern | 2 |
+| shared-state-no-url | core | non-url-sharing | useSharedState for cross-component state without URL | 2 |
+
+Total: 23 failure modes across 6 skills.
+
+## Failure Mode Inventory
+
+### feature-state-hook (9 failure modes)
+| # | Mistake | Priority | Source | Cross-skill? |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)